worlds/tww/docs/fr_The Wind Waker.md

```diff
@@ -0,0 +1,122 @@
+  [Tournoi Saison 7](https://docs.google.com/document/d/1mJj7an-DvpYilwNt-DdlFOy1fz5_NMZaPZvHeIekplc).
+  Ce preréglage contient trois boss requis et une difficulté obscure dur, et même si la liste des options activés peut sembler intimidant, 
+  le préréglage aussi exclue certaines localisations.
+- **Miniblins 2025**: Ce sont (aussi proche que possible) les paramètres utilisés dans le serveur WWR Racing
+  [2025 Season of Miniblins](https://docs.google.com/document/d/19vT68eU6PepD2BD2ZjR9ikElfqs8pXfqQucZ-TcscV8). Ce
+  préréglage est bien si vous êtes nouveau à The Wind Waker! Il n'y a pas beaucoup de localisation dans ce monde, et tu as seulement besoin de compléter deux donjons.
+  Tu commences aussi avec plusieurs objets utiles comme la double magie, une amélioration de capacité pour votre arc et vos bombes et six coeurs.
+- **Mixed Pools**: Ce sont (aussi proche que possible) les paramètres utilisés dans le serveur WWR Racing
+  [Mixed Pools Co-op Tournoi](https://docs.google.com/document/d/1YGPTtEgP978TIi0PUAD792OtZbE2jBQpI8XCAy63qpg). Ce
+  prérégalge contient toutes les entrées randomisés et inclue la plupart des localisations derrière une entrée randomisé. Il y a aussi plusieurs
+  locations de l'overworld, étant donnée que ces paramètres sont censés être joué dans une équipe de deux joueurs. Ce préréglage a aussi six
+  boss requis, mais vu que les pools d'entrées sont randomisés, les boss peuvent être trouvés n'importe où ! Regarder votre carte de l'océan pour
+  déterminer quels îles les boss sont.
+
+## Fonctionnalités planifiées
+
+- Type des coffres Dynamique assorties au contenu en fonction des options activés
+- Implémentation des indices venant du randomiseur de base (options de placement des indices et des types d'indices)
+- Intégration avec le système d'indice d'Archipelago(ex: indices des enchères)
+- Support de l'EnergyLink
+- Logique de la voile rapide en tant qu'option
+- Continuer la correction de bug
+
+## Crédits
+
+Ce randomiseur ne pouvait pas être possible sans l'aide de :
+
+- BigSharkZ: (Dessinateur de l'îcone)
+- Celeste (Maëlle): (correction de logique et de fautes d'orthographe, programmation additionnelle)
+- Chavu: (document sur les difficultés de logique)
+- CrainWWR: (multiworld et assitance sur la mémoire de Dolphin, programmation additionnelle)
+- Cyb3R: (référence pour `TWWClient`)
+- DeamonHunter: (programmation additionnelle)
+- Dev5ter: (Implémentation initiale de l'AP de TWW)
+- Gamma / SageOfMirrors: (programmation additionnelle)
+- LagoLunatic: (randomiseur de base, assistance additionelle)
+- Lunix: (Support Linux, programmation additionnelle)
+- Mysteryem: (Support du tracker, programmation additionnelle)
+- Necrofitz: (documentation additionelle)
+- Ouro: (Support du tracker)
+- tal (matzahTalSoup): (dungeon secrets guide)
+- Tubamann: (programmation additionnelle)
+- mobby45 (traduction du guide français)
```

Comment:
```suggestion
- mobby45 (traduction du guide français)
- Mysteryem: (Support du tracker, programmation additionnelle)
- Necrofitz: (documentation additionelle)
- Ouro: (Support du tracker)
- tal (matzahTalSoup): (guide pour les dungeon secrets)
- Tubamann: (programmation additionnelle)
```
Keep things alphabetical. Also, add yourself in the credits for the English setup guide as well.